### DISCUSSION TITLE
Guys any idea on how to deal with these annoying errors which pop up anytime I open a file in lazyvim?

### DISCUSSION THREAD

**Discussion by shash-786** [LazyVim/LazyVim#4533] [Q&A] [answered]
![Err](https://github.com/user-attachments/assets/f3c69e12-9bdf-49fa-8ef6-2882e29441ce)

Any help is appreciated please and thanks

**Comment by dpetka2001** [comment]
You're probably on an old Neovim version. Update to latest stable 0.10.1 or latest nightly.

**Reply by shash-786** [reply]
![image](https://github.com/user-attachments/assets/46a970a7-63ae-4b0c-819a-ac1cbc91433c)
This is my current version do I need to update still?
I deleted and re-downloaded neovim and lazy vim but still these errors persists

![image](https://github.com/user-attachments/assets/79371574-9b59-4dec-9494-78b52e8c522c)
![image](https://github.com/user-attachments/assets/441a5e8f-1574-445a-b899-6f42943cfe1d)

**Reply by dpetka2001** [reply, marked as answer]
Yes you are on an old Neovim 0.10-dev build. You should update to latest stable 0.10.1 or nightly. Whichever you prefer.

**Reply by shash-786** [reply]
Thanks alot it worked after I moved to latest stable
